tests/system/requests/test_upload.py

```diff
@@ -14,13 +14,13 @@
 
 import base64
 import hashlib
+import http.client
 import io
 import os
+import urllib_parse
```

Comment:
This should be `urllib.parse`.

Reply:
e2b99bd5edf634b21b59178843ae75aaffd27408

Reply:
There's another file where this replacement is needed - `tests/system/requests/test_upload.py`